proposals/0007-access-policy.md

```diff
@@ -0,0 +1,126 @@
+when they are excluded, or of giving the untrustworthy or incompetent power to
+mess things up.
+Therefore, a good policy balances the use of technical and social controls to
+minimize both management overhead and risk to the development process.
+```
+
+## Alternatives Considered
+
+[EIP](https://github.com/ethereum/EIPs) run under two separate levels,
+contributors and editors. Editors are a select group of people that review each
+EIP and have write access. Instead of starting with a select group of editors
+who have exclusive write access, we could begin with an easier way to achieve
+write access and gradually tighten permissions as needed.
+
+[RFCs](https://www.rust-lang.org/governance) have sub-teams that review and
+triage new proposed changes. This is a possibility, but would require more
+granular labeling of different SIMDs.
+
+[PEP](https://peps.python.org/pep-0013/) has an election of a council that
+reviews and triages new proposed changes. The PEP approach may result in an
+increase in governance overhead.
+
+## New Terminology
+
+Core Contributor - Individuals with Core access to SIMDs that can assist in
+triaging different SIMDs.
+
+## Detailed Design
+
+### Level 1 - Triage Access
+
+Requirements: One voucher from any user with level 2 or above access.
+
+Contributors with triage access will have the associated [triage Github access
+policy](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)
+for the [Solana Improvement
+Documents](https://github.com/solana-foundation/solana-improvement-documents)
+repository. They will also have the Contributor role in the Solana Tech
+Discord. This is the lowest level of access.
+
+### Level 2 - Core Access
+
+Requirements: One voucher from anyone with level 3 access or two vouchers from
+anyone from level 2.
+
+Core contributors will have the associated [write Github access
+policy](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)
+for the [Solana Improvement
+Documents](https://github.com/solana-foundation/solana-improvement-documents)
+repository. They will also have the Core Contributor role in the Solana Tech
+Discord.
+
+Most notably these permissions allow the contributor to approve and merge pull
+requests.
+
+### Level 3 - Maintain Access
+
+Requirements: One voucher from anyone with level 3 access.
+
+Contributors will have the associated [maintain Github access
+policy](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)
+for the [Solana Improvement
+Documents](https://github.com/solana-foundation/solana-improvement-documents)
+repository. They will also have the Contributor role in the Solana Tech
```

Comment:
Don't think role in discord matters as much for those with maintain access.

Reply:
I feel the Solana Tech Discord roles are an independent matter entirely, which also needs a formal ACL process of course.

Reply:
Should the Solana Tech Discord only be concerned with the Solana Labs validator implementation?

Reply:
I do think there's a discussion to be had about what Solana Tech Discord and its roles ought to morph into as we clarify the lines between the various entities. New channels could perhaps be created for discussing proposals, for folks that prefer to discuss some points outside of github issues.

The "Core Contributor role" in the Solana Tech Discord essentially means write access to the solana-labs/solana monorepo and the SPL repo right now. Until Solana Tech Discord is cleaned up, I think granting that role to people who achieve write access here will be making the problem worse.